modules/ROOT/pages/tokenization-example.adoc

```diff
@@ -97,17 +97,16 @@ image::tokenization-example-edit-token-service.png[]
 . Confirm the Runtime Fabric assignment. +
 The tokenization service name is “mytoken1” and the implementation URL will be: “https://mytoken1­tokenizer:3443”.
 
-The service is available via HTTPS only. The tokenization service is hardcoded to listen on port 3443. The hostname portion is formed by taking the “Service Name” + “tokenizer” to arrive at the Kubernetes service name. The service name in this example is “mytoken1”.
+The service is available through HTTPS only. The tokenization service is hardcoded to listen on port 3443. The hostname portion is formed by taking the “Service Name” + “tokenizer” to arrive at the Kubernetes service name. The service name in this example is “mytoken1”, so the hostname is "mytoken1tokenizer".
```

Comment:
hard coded

Reply:
or maybe remove that jargon entirely. The tokenization service listens on port 3443.